debian/control

```diff
@@ -0,0 +1,17 @@
+Source: ooniprobe-cli
+Maintainer: Federico Ceratto <federico@openobservatory.org>
+Section: devel
+Priority: optional
+Build-Depends: dpkg-dev, debhelper-compat (= 12)
+Standards-Version: 4.5.0
+Vcs-Browser: https://salsa.debian.org/go-team/packages/probe-cli
+Vcs-Git: https://salsa.debian.org/go-team/packages/probe-cli.git
+Homepage: https://github.com/ooni/probe-cli
+Rules-Requires-Root: no
+XS-Go-Import-Path: github.com/ooni/probe-cli
+
+Package: ooniprobe-cli
+Architecture: any
+Depends: ${misc:Depends}, ${shlibs:Depends}
+Built-Using: ${misc:Built-Using}
+Description: Next generation OONI Probe CLI
```

Comment:
I think we can omit "next generation" since it's not so useful for users?

Reply:
I guess that was taken from the github description field.

Maybe we should agree on what we use as a description for the package everywhere and @agrabeli can maybe help with this copy.